lib/http-bridge/src/Cardano/Wallet/HttpBridge/Compatibility.hs

```diff
@@ -111,6 +125,32 @@ instance KeyToAddress (HttpBridge 'Testnet) where
 instance KeyToAddress (HttpBridge 'Mainnet) where
     keyToAddress = keyToAddressWith emptyAttributes
 
+instance KeyToAddress t => WalletKey t SeqKey where
+    keyToAddress2 = keyToAddress @t . unSeqKey
+    changePassphrase old new = SeqKey . changePassphraseSeq old new . unSeqKey
+    unwrapKey = unSeqKey
+
+instance WalletKey (HttpBridge 'Mainnet) RndKey where
+    keyToAddress2 (RndKey rootKey addrKey accountIndex addressIndex) =
+        keyToAddressWith
+        (rndToAddressAttrs addrKey (publicKey rootKey) accountIndex addressIndex)
+        addrKey
+
+instance WalletKey (HttpBridge 'Testnet) RndKey where
+    keyToAddress2 (RndKey rootKey addrKey accountIndex addressIndex) =
+        keyToAddressWith
+        (rndToAddressAttrsWithProtocolMagic addrKey (publicKey rootKey) accountIndex addressIndex)
+        addrKey
+
+rndToAddressAttrs, rndToAddressAttrsWithProtocolMagic
+    :: Key 'AddressK XPub
+    -> Key 'RootK XPub
```

Comment:
just nitpick, but maybe we want `Key 'RootK XPub` to be first argument ...

Reply:
True, changed.